pkg/scheduler/framework/plugins/interpodaffinity/filtering_test.go

```diff
@@ -22,7 +22,7 @@ import (
 	"strings"
 	"testing"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Revert unnecessary change

Reply:
This seems to be done by the VS Code go plugin, let me revert this